src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -328,6 +326,23 @@ public async Task Accept(Guid orgId, Guid organizationUserId, [FromBody] Organiz
         }
     }
 
+    private async Task<bool> ShouldHandleResetPasswordAsync(Guid orgId)
+    {
+        var organization = await _organizationRepository.GetByIdAsync(orgId);
+
+        if (organization is not { UsePolicies: true })
+        {
+            return false;
+        }
```

Comment:
Please use `IApplicationCacheService`. Example: 

```cs
        var orgAbility = await _applicationCacheService.GetOrganizationAbilityAsync(orgId);
        if (orgAbility is not { UsePolicies: true })
```

This uses an in-memory cache synced between instances using a message bus.